### ISSUE TITLE
Feature request: print html tag of the node (not including its children) for lexbor engine

### ISSUE THREAD

**Issue by gshashank84** [rushter/selectolax#64]
To check if two nodes are equal, we try to compare the html tag of them. But here the tag of its children also comes in the output. Please add an method that print html string of single node only (i.e. not including its parents).

**Comment by gshashank84** [comment]
Also can we make the __eq__ method of LexborNode performant? As we are internally comparing html of the nodes for the equality operator, it takes huge computation time.

**Comment by rushter** [comment]
> Also can we make the `__eq__` method of LexborNode performant? As we are internally comparing html of the nodes for the equality operator, it takes huge computation time for comparing two nodes of a big tree DOM.

Yeah, this is an old problem, but there is no easy way to solve it, since lexbor/modest engines do not have internal IDs or something like that to simply compare them. We could compare their location in memory, but we still need to perform a big check in case of a miss.
